airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -487,6 +488,7 @@ def __init__(
         self._message_repository = message_repository or InMemoryMessageRepository(
             self._evaluate_log_level(emit_connector_builder_messages)
         )
+        self._state_manager = state_manager
```

Comment:
My guess is that this makes sense here given that we want to instantiate cursors with the state eventually to avoid a `set_initial_state` method call